### PR TITLE
Add BGRA pixel format

### PR DESCRIPTION
**Pull request by cloudhead**

### CODE REVIEW

File: luminance/src/pixel.rs [hunk 1]
```diff
@@ -880,6 +884,20 @@ impl_Pixel!(
...
+
+impl_Pixel!(
+  NormBGRA8UI,
+  (u8, u8, u8, u8),
+  u8,
+  NormUnsigned,
+  Format::BGRA(Size::Eight, Size::Eight, Size::Eight, Size::Eight)
+);
+impl_ColorPixel!(NormBGRA8UI);
+impl_RenderablePixel!(NormBGRA8UI);
```
Comment (phaazon) [inline luminance/src/pixel.rs]: Unfortunately, it’s not renderable. 🙁 

Source:
> [An internal format iscolor-renderableif it isRED,RG,RGB,RGBA](https://www.khronos.org/registry/OpenGL/specs/gl/glspec46.core.pdf)

Comment (cloudhead) [reply luminance/src/pixel.rs]: That explains it ;(

**Comment by cloudhead** [comment]
Closing as I don’t think this is so useful if it’s not renderable.

**Comment by phaazon** [comment]
It could be useful for people who wants to load textures in BGRA, but so far no one complained (and I don’t use that format). I might add them later then. Thanks anyway for the time you passed on it!

**Pull request closed without merging**